.packit.yaml

```
@@ -22,6 +22,15 @@ jobs:
     - epel-9
     enable_net: False
 
+- job: tests
+  trigger: pull_request
+  metadata:
+    targets:
+    - fedora-latest-stable
+    - epel-8
```

This one will collide with the `fedora-all` from the previous job. (So the status will be overwritten. You can use `identifier` to fix that. The value can be anything.)

```suggestion
    identifier: "internal"
    targets:
    - fedora-latest-stable
    - epel-8
```